docs/send-data/collect-from-other-data-sources/azure-blob-storage/collect-logs-azure-blob-storage.md

```diff
@@ -1,10 +1,10 @@
 ---
 id: collect-logs-azure-blob-storage
 title: Collect Logs from Azure Blob Storage
-sidebar_label: Collecting Metrics
+sidebar_label: Collecting Logs
 description: Instructions for configuring a pipeline for shipping logs available from Azure Blob Storage to an Event Hub, on to an Azure Function, and finally to an HTTP source on an hosted collector in Sumo Logic.
```

Comment:
```suggestion
description: Instructions for configuring a pipeline for shipping logs available from Azure Blob Storage to an Event Hub, on to an Azure Function, and finally to an HTTP source on a hosted collector in Sumo Logic.
```

Reply:
These are technical routes of data flow and not grammatical lines of instructions. They should be how they are. You shouldn't be changing these.

Reply:
The description should be crisp and good enough to let users know what they will be seeing in the document. Detailed info can be given in the document.

Reply:
Only updating pronouns here. Otherwise, it is absolutely crisp and accurate. I object any other changes here.

Reply:
I understand your preference to keep the technical route information and we are already taken care of that in the document below.
Both the existing content and the suggested changes effectively convey the instructions for configuring the pipeline, we can accommodate your preference if it improves readability. Ultimately, our main focus is to ensure that the instructions are clear, accurate, and easy to follow for our audience. It's just that the suggested changes eliminate the repetition of the phrase "on to" and maintains the overall meaning and context of the instructions.

Reply:
See, the entire pipeline is a step wise process. What happens is, the data flows from a storage account to an event hub. Thats step 1, then the data from the event hub flows to the azure functions, thats step2. And then the data flows from the azure function to the http source.
What on to means is simply this '->'. Data from SA->EventHub->AzureFunc->HTTPSource. It cannot be in any other other. This defined as the pipeline of data flow.
What you are suggesting is we remove the on to keywords and just mention the resouces, what will that mean?
That will mean Data from SA,EventHub,AzureFunc,HTTPSource. What do you as a layman understand from this? The pipeline can now be in any order, which is not correct. Do you understand what I am trying to say here? I could show you a demo of what the pipeline does if you need to more clarity.

Reply:
Why will i provide incorrect information? What you are trying to say is what will be more confusing for the audience instead.

Reply:
Everything is not about readability and removing repetition. It should be about what is exactly accurate. It is not about being slightly accurate or slightly better, it should be exactly accurate. That's what it is.

Reply:
@JKashyap96 Can we just add starting and ending point of the log instead of explaining the complete flow? Something like this : 'Instructions for configuring a pipeline for shipping logs available from Azure Blob Storage to a HTTP source on a hosted collector in Sumo Logic.'

Reply:
@JKashyap96 : I understand where you coming from. I insist you to understand what I am suggesting here. We are talking about readability here and crisp sentences without changing the meaning of it. 
Either you should go with :
Instructions for configuring a pipeline for shipping logs available from Azure Blob Storage to an Event Hub, Azure Function, and HTTP source on a hosted collector in Sumo Logic.
or
Configure a pipeline to ship logs from Azure Blob Storage to an Event Hub, Azure Function, and HTTP source on a hosted collector in Sumo Logic." This can also go in the description which is not changing the meaning.

cc: @jpipkin1 : Would love to have your suggestion here?

Reply:
Both of these are incorrect technically.
We go for this: 'Instructions for configuring a pipeline for shipping logs available from Azure Blob Storage to a HTTP source on a hosted collector in Sumo Logic.'

Reply:
This is better, we can go with this.

Reply:
> @JainM6, I like your second suggestion.

Thanks @jpipkin1 for reviewing this one. I think this one would go "Configure a pipeline for shipping logs available from the Azure Blob Storage to an HTTP source on a hosted collector in Sumo Logic" as it conveys that there is a flow that will be described in the doc. What do you suggest?

Reply:
Configure a pipeline to ship logs from the Azure Blob Storage to an HTTP source on a hosted collector in Sumo Logic.